Makefile

```
@@ -47,12 +47,21 @@ BUILD_FLAGS := -tags "$(build_tags)" -ldflags '$(ldflags)'
 ###                                  Build                                  ###
 ###############################################################################
 
-build:
+MODULES := . e2e
+
+tidy:
+	@cp go.work.example go.work
+	@for module in $(MODULES); do \
+		(cd $$module && go mod tidy); \
+	done
+	@go work use && go work sync
```

Also, we have modules defined both here and in the `go.work.example`, which can be not in sync. If we want to use this approach, I would suggest to keep an empty `go.work` (`use()`), and define the modules only in the Makefile:

```suggestion
	@cp go.work.example go.work
	@for module in $(MODULES); do \
		go work use $$module; \
		(cd $$module && go mod tidy); \
	done
	@go work use && go work sync
```